### PR TITLE
Feature/cleanup printer manager

### LINKED ISSUE #548: Smaller screen printer table wrapping incorrectly
### Description of the problem

```bash
The screenshot added shows some unfortunate wrapping. The buttons should take up the space they need: flex.
```

### Reproduction of the problem

```bash
Set to slightly smaller screen
```

image

### LINKED ISSUE #625: Template printer management wrong tags
### Description of the problem

```bash
Some small/span not closed, opened or in wrong order.
HTML is quite dev friendly, but you never know.

Cleanup time
```

### Reproduction of the problem

```bash
...
```

### PR DESCRIPTION
**Pull request by NotExpectedYet**
This splits up the printer manage ejs into chunks. 

Cleans up some id's and namings. 

This doesn't touch the js as required for #612 but lays the ground work for it. Will do that in another PR. 

Also to note, farmStatus card doesn't work yet. Will add the JS for that in the #612 PR, also may change styling just put some buttons in for now. 

Fixed #625 - incorrect printer manager tags
Fixed #548 - Smaller screen aciton buttons wrapped incorrectly

### CODE REVIEW

**Comment by davidzwa** [comment]
Awesome stuff! Will take a look in an hour or 1.5, but dont have too long to check

**Comment by NotExpectedYet** [comment]
> Awesome stuff! Will take a look in an hour or 1.5, but dont have too long to check

No rush, I can branch off of this to start the printerManagerRunner break up anyway.

File: CHANGELOG.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ All notable changes to this project will be documented in this file.
     - Fixed #641: Opening the console on the Printers Page with offline printers would crash the browser due to spam.
     - Fixed #638: Fixed login not working anymore after refactor
     - Fixed `snapshots` instead of `snapshot` bug on client system Javascript bundle
+    - Fixed #625 - Incorrect html tags on Printer Manager
+    - Fixed #548 - Smaller screen aciton buttons wrapped incorrectly on Printer Manager
```
Comment (davidzwa) [inline CHANGELOG.md:48]: typo action

**Comment by davidzwa** [comment]
Love the refactor in ejs partials. Would be an absolute yes from me on that end. The buttons could even be automated with NodeJS code reducing the code footprint further (same goes for dashboard, which were my plans for later). Very nice work indeed!

The new cards are very organized. Two points of feedback.
1) The printer logs are now 5 lines. With 40 printers the amount of scrolling is quite high. Alternatives: group buttons to filter message types and printers in the card title/header. Second alt: 10 lines. Third alt: a button to open a modal with 50 lines of logs shown.
1) The spinner on the bottom of the logs doesn't disappear (I applied the SSE fix manually beforehand)
1) The Farm Status: this printers page is meant for studying & adjusting specific printers. If I'd go for a total overview I'd visit the dashboard. Would be a no from me.
1) The table - `Printer` and `OctoPrint` columns contain information which is not compact nor are they necessary all the time making the printer rows large for a bigger farm using this page on a day to day basis. I would defer this to a tooltip, expandable row or button opening a modal. That way the information is visible by user choice.

### Example visible on PR #527 
I adjusted the printer map branch to look like so - the first row is what I like, the second is a bit too compact.
image
- I would even go further and remove the `Firmware:`, `Model:` and `RaspberryPi:` words. These are repeated a lot causing wrapping on smaller screens.
- The plugin update and OP update buttons tend to wrap => I would like to have buttons without text for them + mouse hover tooltips. And as stated before truly like the idea of expandable rows. Ive noticed in my app that users really appreciate that on a page with high information density.

**Review by davidzwa — CHANGES_REQUESTED** [review]
As stated I see some points of improvement (spinner not disappearing) and some points of discussion. Lets talk!

**Comment by NotExpectedYet** [comment]
> Love the refactor in ejs partials. Would be an absolute yes from me on that end. The buttons could even be automated with NodeJS code reducing the code footprint further (same goes for dashboard, which were my plans for later). Very nice work indeed!
> 
Thanks, yeah did consider doing the buttons as well but like you say PR for later. 

> The new cards are very organized. Two points of feedback.
> 
>     1. The printer logs are now 5 lines. With 40 printers the amount of scrolling is quite high. Alternatives: group buttons to filter message types and printers in the card title/header. Second alt: 10 lines. Third alt: a button to open a modal with 50 lines of logs shown.
> 
Will fix, wasn't suppose to be left like that. The buttons are outside of the scope of this PR really but will keep in minde for when I tackle the JS. I dunno about the pop out as you can already view the entire printer interaction in the printer logs. Would someone not investigate a printer issue there? if at all. My experience is neither tbh xD.

>     2. The spinner on the bottom of the logs doesn't disappear (I applied the SSE fix manually beforehand)
> 

I will fix that but it's re-doing the JS and I was going to do that in another PR. 

>     3. The Farm Status: this printers page is meant for studying & adjusting specific printers. If I'd go for a total overview I'd visit the dashboard. Would be a no from me - can explain why in DM.
> 

Yes but it's also got the bulk action buttons in place currently so the page is dual purpose until it get's split. It's just a break down on your current printer states, and doubt I'll be keeping it as it looks there it's just a place holder to add something when I do the JS. I think it's handy having a quick overview here without having to go to the dashboard view. I can quickly see how many I have online/whether I need to bring others online. 

>     4. The table - `Printer` and `OctoPrint` columns contain information which is not compact nor are they necessary all the time making the printer rows large for a bigger farm using this page on a day to day basis. I would defer this to a tooltip, expandable row or button opening a modal. That way the information is visible by user choice.
> 
Nobody reads the tool tips, so I'd rather not. They never do for the buttons and exactly why I leave text in there.
It was a user requested feature for this page so I'm hesitant to remove the OctoPrint one. I agree the printer stuff is pointless been there and needs moving to the settings modal/combine the printer statistics into a Printer information panel. 

> 
> ### Example visible on PR #527
> 
> I adjusted the printer map branch to look like so - the first row is what I like, the second is a bit too compact.
> image
> 
>     * I would even go further and remove the `Firmware:`, `Model:` and `RaspberryPi:` words. These are repeated a lot causing wrapping on smaller screens.
> 
Yeah these can go. 

>     * The plugin update and OP update buttons tend to wrap => I would like to have buttons without text for them + mouse hover tooltips. And as stated before truly like the idea of expandable rows. Ive noticed in my app that users really appreciate that on a page with high information density.

I'll remove the text from the OP/Plugin updates on that yeah agree. There's buttons at the top which I gather are used more anyway.

**Comment by NotExpectedYet** [comment]
Forgot to add the dropdowns are a good idea, will so those.

**Review by davidzwa — APPROVED** [review]
With the changes on their way, I find this a massive improvement. Nice!
Logically didnt test, as Im currently reviewing on mobile.

**Comment by NotExpectedYet** [comment]
Screeny, removed the text as well from the plugin/op updates. Gather the bulk ones supersede them for users anyway so they're are probably more just notifications. The rest, is for the JS tackle. 

image

**Pull request merged**